src/rnanorm/command.py

```diff
@@ -21,27 +21,35 @@ def main():
     """
     )
     parser.add_argument(
-        "expressions_file",
-        type=str,
-        help="tab-delimited file with gene expression data (genes in rows, samples in cols)",
+        "expression", type=str, help="tab-delimited file with gene expression data (genes in rows, samples in cols)",
     )
-    parser.add_argument("gene_lengths_file", type=str, help="tab-delimited file with gene lengths")
-    parser.add_argument("--output", "-o", type=str, help="output file name")
+    parser.add_argument("--gene-lengths", type=str, help="tab-delimited file with gene lengths")
+    parser.add_argument("--tpm-output", type=str, help="TPM output file name")
+    parser.add_argument("--cpm-output", type=str, help="CPM output file name")
 
     args = parser.parse_args()
-    expressions_path = args.expressions_file
-    gene_lengths_path = args.gene_lengths_file
-    output_path = args.output
+    expression_path = args.expression
+    gene_lengths_path = args.gene_lengths
+    tpm_output_path = args.tpm_output
+    cpm_output_path = args.cpm_output
 
-    if not os.path.exists(expressions_path):
-        print(f"File not found {expressions_path}")
+    if not os.path.exists(expression_path):
+        print(f"Expressions file not found {expression_path}")
         exit(1)
 
-    if not os.path.exists(gene_lengths_path):
-        print(f"File not found {gene_lengths_path}")
-        exit(1)
+    if tpm_output_path:
+        if gene_lengths_path is None:
+            print("--gene-lengths must be given for --tpm-output")
+            exit(1)
+
+        if not os.path.exists(gene_lengths_path):
+            print(f"Gene lengths file not found {gene_lengths_path}")
```

Comment:
You are mixing f-strings and `format` in the codebase. It would be nice to unify the code.

Reply:
done